test/testOpenAPIFile.js

```diff
@@ -0,0 +1,300 @@
+var path = require("path");
+var fs = require("fs");
+
+if (!OpenAPIFile) {
+	var OpenAPIFile = require("../OpenAPIFile");
+	var OpenAPIFileType = require("../OpenAPIFileType");
+	var JsonFileType = require("ilib-loctool-json/JsonFileType");
+
+	var CustomProject = require("loctool/lib/CustomProject.js");
+	var TranslationSet =  require("loctool/lib/TranslationSet.js");
+	var ResourceString =  require("loctool/lib/ResourceString.js");
+	// var ResourcePlural =  require("loctool/lib/ResourcePlural.js");
+	// var ResourceArray =  require("loctool/lib/ResourceArray.js");
+}
+
+var p = new CustomProject({
+	name: 'foo',
+	id: 'foo',
+	sourceLocale: 'en-US'
+}, './test/testfiles', {
+	locales:['ru-RU'],
+	targetDir: '.',
+	nopseudo: true,
+	openapi: {
+		schemas: ['./test/testfiles/openapi-schema.json'],
+		mappings: {
+
```

Comment:
Without mappings, we don't apply the json schema to the file. Instead, the JsonFile instance applies the generic schema that is tested below.

Reply:
You're right! It was using default schema from json plugin. Added correct mapping and verified in tests that it's using OpenAPI schema now.